photon-lib/src/main/scala/com/linkedin/photon/ml/stat/BasicStatisticalSummary.scala

```diff
@@ -43,7 +43,8 @@ case class BasicStatisticalSummary(
     min: BreezeVector[Double],
     normL1: BreezeVector[Double],
     normL2: BreezeVector[Double],
-    meanAbs: BreezeVector[Double])
+    meanAbs: BreezeVector[Double],
+    interceptIndex: Option[Int])
```

Comment:
I think it makes sense to package it up this way, but the name `BasicStatisticalSummary` doesn't really make sense anymore. It's more like `DataSummary` or something more general like that.

Reply:
Going to take care of this in a separate PR.